src/app/review-item/components/review-page/review-page.component.ts

```diff
@@ -91,9 +93,15 @@ export class ReviewPageComponent implements OnInit {
     private snackBar: MatSnackBar,
     private loader: LoaderService,
     private router: Router,
-    private factCheckService: FactCheckService
+    private factCheckService: FactCheckService,
+    private formBuilder: FormBuilder
   ) {
     this.routerState = this.router.getCurrentNavigation().extras?.state;
+
+    this.reviewForm = this.formBuilder.group({
+      legal: [false, Validators.requiredTrue],
+      policy: [false, Validators.requiredTrue]
+    });
```

Comment:
Coulnd't you create all FormControls for the questions in here as well and then pass down only the FormControl to the QuestionsComponent? 
That way you would not have to pass down the entire parentFormGroup.

You could also omit the EventEmitter by listening to changes on the FormControl inside of this component.

Reply:
Yes, this could be done, I think. Probably a custom validator has to be implemented because of the conditional child questions. I will have a look at it...

Reply:
Tried to change the implementation, but without refactoring the parent-child-logic for questions (which is implemented in QuestionComponent) you end up with creating or passing form controls for child questions. Without refactoring the current behavior, the implementation will be more complex than it is now IMHO. 😕

Reply:
Alright, I guess then just leave it like that for now 👍